flags.go

```diff
@@ -14,6 +14,7 @@ type FlagValue interface {
 	HasChanged() bool
 	Name() string
 	ValueString() string
+	ValueStringArray() []string
```

Comment:
This is a breaking change in the interface.

Reply:
Yep.  This is dependent on https://github.com/spf13/pflag/pull/181 being merged first.

Reply:
I mean it's a breaking change for anyone using custom flag types and for the 1.x version there is a backward compatibility promise.